### PR TITLE
Fix: @EntryPoint completion inserts two @ characters

### LINKED ISSUE #812: `@EntryPoint` completion inserts two `@` characters
In the Q# Playground or in VS Code, typing `@` in a namespace correctly brings up the `@EntryPoint()` attribute, but committing the completion results in `@@EntryPoint()`.

The insertion range probably needs to consume the `@` before the cursor, which is something I missed initially.

Seen as of a153981617a6c8a2bb745a8d15176755d399af44

### PR DESCRIPTION
**Pull request by minestarks**
Fixes #812

Good news: This was a one line fix
Bad news: That one line was a regex

### CODE REVIEW

File: vscode/language-configuration.json [hunk 1]
```diff
@@ -12,12 +12,29 @@
   "autoClosingPairs": [
     ["[", "]"],
     ["(", ")"],
-    ["{", "}"]
+    ["{", "}"],
+    { "open": "\"", "close": "\"", "notIn": ["string"] }
```
Comment (minestarks) [inline vscode/language-configuration.json:16]: Unrelated fix: Noticed we were missing quotes in the auto-close list.

**Review by ScottCarda-MS — APPROVED** [review]
